prime-dotnet-webapi/Configuration/EnrolleeConfiguration.cs

```diff
@@ -9,10 +9,13 @@ public class EnrolleeConfiguration : IEntityTypeConfiguration<Enrollee>
         public void Configure(EntityTypeBuilder<Enrollee> builder)
         {
             builder
-                .HasKey(a => new { a.Id });
```

Comment:
Dont need the key in the configuration, we have the [Key] attribute on the model